src/javascripts/ng-admin/Crud/button/CsvExportConfiguration.js

```diff
@@ -0,0 +1,26 @@
+define(function () {
```

Comment:
We're not using requirejs anymore, but webpack. Please leave the `define` aside and use `export` instead.